src/dispatching/dispatcher.rs

```diff
@@ -280,6 +302,92 @@ where
     }
 }
 
+const WORKER_QUEUE_SIZE: usize = 64;
+
+fn spawn_worker<Err>(
+    deps: DependencyMap,
+    handler: Arc<UpdateHandler<Err>>,
+    default_handler: Arc<DefaultHandler>,
+    error_handler: Arc<dyn ErrorHandler<Err> + Send + Sync>,
+) -> WorkerTx
+where
+    Err: Send + Sync + 'static,
+{
+    let (tx, rx) = tokio::sync::mpsc::channel(WORKER_QUEUE_SIZE);
+
+    let deps = Arc::new(deps);
+
+    tokio::spawn(async move {
```

Comment:
`async move { fut.await }` shouldn't be necessary here? you should be able to spawn the `.for_each()` directly

Reply:
Fixed.